README.md

```diff
@@ -22,6 +22,17 @@
 go get github.com/libp2p/go-libp2p-kad-dht
 ```
 
+## Running Tests
+
+This package depends on tests that require crypto keys too large to inline into
+a peer ID. These tests generate 512bit RSA keys for speed purposes. To run
+these tests successfully, you must set the environment variable
+`LIBP2P_ALLOW_WEAK_RSA_KEYS` to any non-empty value, e.g. `1`.
+
+```bash
+$ LIBP2P_ALLOW_WEAK_RSA_KEYS=1 go test
```

Comment:
We can't do this. We should either just use ed25519, generate full RSA keys for the tests that need them, or pre-generate keys.

Reply:
k i actually had it like that earlier. will do 2048. only increases test execution by 0.6s on my local machine.

Reply:
That's probably fine. It's only really an issue for _massive_ test suites (e.g., sharness) but we can set environment variables internally there.